docs/getting-started/croeseid-testnet.md

```diff
@@ -2,13 +2,12 @@
 
 The latest Crypto.org Chain Testnet has been named as **Croeseid**.
 
-This is a detailed documentation for setting up a Validator or a full node on Crypto.org Croeseid testnet.
+This is a detailed documentation for setting up a Validator or a full node on Crypto.org Croeseid testnet `testnet-croeseid-3`.
 
 ## Pre-requisites
 
 **Remarks**:
-Please follow this [guide](https://github.com/crypto-com/testnets/tree/main/testnet-croeseid-2#network-upgrade-guide). If you are upgrading from `v0.8.1` to `v0.9.1`
-
+`testnet-croeseid-3` is the latest Crypto.org Chain testnet. If you have joined `testnet-croeseid-2` and received test token before, you should be able to access the test token with the same key. Otherwise, you can request test token by sending send a message on [Discord](https://discord.gg/pahqHz26q4) **#request-tcro** channel.
```

Comment:
```suggestion
`testnet-croeseid-3` is the latest Crypto.org Chain testnet. If you have joined `testnet-croeseid-2` and received test tokens before, you should be able to access the test token with the same key. Otherwise, you can request test tokens by sending a message on [Discord](https://discord.gg/pahqHz26q4) **#request-tcro** channel.
```